integration/networktest/tests/ci/full_network_sim_test.go

```diff
@@ -0,0 +1,18 @@
+package ci
+
+import (
+	"testing"
+	"time"
+
+	"github.com/obscuronet/go-obscuro/integration/networktest"
+	"github.com/obscuronet/go-obscuro/integration/networktest/env"
+	"github.com/obscuronet/go-obscuro/integration/networktest/traffic"
+)
+
+// TestFullNetworkSim reimplements the test case of our current "sim" tests (the ERC20 transaction injector)
+func TestFullNetworkSim(t *testing.T) {
```

Comment:
Should this be removed until the sims are replaced ?

Reply:
Yeah probably, I'll get rid of the erc20 placeholder too.